source/nodejs/adaptivecards-designer-app/dist/containers/bf-image-container.css

```diff
@@ -153,4 +153,103 @@
 
 .ac-inlineActionButton:active {
     background-color: #CCCCCC;
-}
\ No newline at end of file
+}
+
+/* Popup menu */
+
+.ac-ctrl-overlay {
+    position: absolute;
+    left: 0;
+    top: 0;
+    z-index: 10000;
+}
+  
+@keyframes fadeIn {
+    0% {
+        opacity: 0;
+    }
+}
+  
+@keyframes slideTopToBottom {
+    0% {
+        transform: translate(0, -20px);
+    }
+}
+  
+@keyframes slideBottomToTop {
+    0% {
+        transform: translate(0, 20px);
+    }
+}
+  
+@keyframes slideLeftToRight {
+    0% {
+        transform: translate(-20px, 0);
+    }
+}
+  
+@keyframes slideRightToLeft {
+    0% {
+        transform: translate(20px, 0);
+    }
+}
+  
+.ac-ctrl-slide {
+    animation-duration: 0.5s;
+    animation-timing-function: cubic-bezier(0.1, 0.9, 0.2, 1);
+}
+  
+.ac-ctrl-slide.ac-ctrl-slideTopToBottom {
+    animation-name: fadeIn, slideTopToBottom;
+}
+  
+.ac-ctrl-slide.ac-ctrl-slideBottomToTop {
+    animation-name: fadeIn, slideBottomToTop;
+}
+  
+.ac-ctrl-slide.ac-ctrl-slideLeftToRight {
+    animation-name: fadeIn, slideLeftToRight;
+}
+  
+.ac-ctrl-slide.ac-ctrl-slideRightToLeft {
+    animation-name: fadeIn, slideRightToLeft;
+}
+  
+.ac-ctrl {
+    box-sizing: border-box;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 14px;
+    font-weight: normal;
+}
+  
+.ac-ctrl-dropdown-item {
+    width: 100%;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+    display: block;
+    padding: 0px 12px 0px 12px;
+    line-height: 30px;
+    cursor: pointer;
+}
+  
+.ac-ctrl-dropdown-item:hover {
+    background-color: #EEEEEE;
+}
+  
+.ac-ctrl-dropdown-item:focus {
+    outline: 0;
+    background-color: #CCCCCC;
+}
+  
+.ac-ctrl-popup-container {
+    border: 1px solid #EEEEEE;
+    background-color: white;
+    position: absolute;
+    box-shadow: 0 0 15px -5px rgba(0, 0, 0, 0.4);
+    overflow-y: auto;
+}
```

Comment:
should double-check that these classes have good behavior in High Contrast (here and elsewhere)

Reply:
No effort was made here. These classes are copied straight from the designer (with a different prefix)